core/templates/domain/story/StoryContentsObjectFactory.ts

```diff
@@ -72,18 +76,18 @@ export class StoryContents {
         return nodes[i].getId();
       }
     }
-    return null;
+    return '';
```

Comment:
Why change this to `''`?

Reply:
changed the return type to `string | null` instead

Reply:
PTAL at this method, it is redundant and not used anywhere.Should I remove this one?

Reply:
If it is not needed the remove it.

Reply:
done